robot_skills/src/robot_skills/util/entity.py

```diff
@@ -88,12 +89,12 @@ def is_a(self, super_type):
     @property
     def pose(self):
         """ Returns the pose (at this point, as a geometry_msgs.Pose, in the future as a kdl frame """
```

Comment:
I'll grep for this at some point, because there should not be a need for this any more.